website/docs/core-concepts/stacks/define-components.mdx

```diff
@@ -212,3 +212,21 @@ There are two types of components:
   <dt>`abstract`</dt>
   <dd>An `abstract` component is more like a blueprint. It can’t be deployed on its own. Instead, it’s a base configuration that needs to be extended or inherited by other components. This is similar to an ["abstract base classes"](https://en.wikipedia.org/wiki/Abstract_type) in programming—it defines reusable configurations, but it’s not complete enough to be deployed directly.</dd>
 </dl>
+
+### Skipping Components with `metadata.enabled`
+
+The `metadata.enabled` parameter allows for conditional inclusion of components. By default, components are enabled. When `metadata.enabled` is set to `false`, the component is completely skipped during deployment - no workspace is created and no Terraform commands are executed. This differs from `vars.enabled: false`, which still processes the component but may result in resource deletion. This feature is particularly useful for temporarily disabling components without removing their configurations.
```

Comment:
I think we need to clarify that vars.enabled is a Cloud Posse convention when designing components. We implement in our components and modules an enabled feature flag, that toggles whether or not the resources are created or destroyed. This has nothing to do with atmos and atmos behavior is not controlled by any vars.

Reply:
commited your suggestion